### DISCUSSION TITLE
Inference of a 100T Model

### DISCUSSION THREAD

**Discussion by BetaDoggo** [turboderp/exllamav2#278] [General]
With recent discussions of real time frankenmerges by running passes over layers multiple times, I was wondering what the limit would be in terms of size. Could it theoretically scale to 100T and beyond? If speed scales linearly with the number of layers, I think it would only take around 50s per token on a 3090. Is this how we achieve ASI?

**Comment by turboderp** [comment]
100 trillion parameters?
